tools/test-api

```diff
@@ -106,5 +110,16 @@ with test_server_running(
     )
     test_invalid_api_key(client)
 
+    # Test account deactivated error
+    guest_user.is_active = False
+    guest_user.save()
```

Comment:
This should be `do_deactivate_user`, I think.

Reply:
used `change_user_is_active` from actions.py along with a comment
```python
# we deactivate user manually because do_deactivate_user removes user session
```
as this is what we do in most of tests in `test_decorators.py` for deactivating users in tests.